client/src/panels/panel_declarations/finances/drr_dp_resources/dp_rev_split.js

```diff
@@ -22,6 +24,14 @@ const special_cols = _.flatMap(planning_years, (year) => [
 ]);
 const dp_cols = [...planning_years, ...special_cols];
 
+const spending = (value) => (
+  <Format style={{ color: "#008000" }} type={"dollar"} content={value} />
```

Comment:
you can throw these in `infobase_color_constants`

Reply:
Actually now that I think of it, green and red colors actually are already defined. Check `color_defs.js`, you can get these from `window. infobase_color_constants`

Reply:
The green in color_defs.js were too low contrast, so I took the green and red from the estimates comparison